src/main/java/org/openrewrite/java/dependencies/internal/VersionParser.java

```diff
@@ -29,7 +27,7 @@ public class VersionParser {
     public VersionParser() {
     }
 
-    public @Nullable Version transform(String original) {
+    public Version transform(String original) {
```

Comment:
Read the parse function below, there is only 1 return statement: `new DefaultVersion`. So this method returns absolutely a non-null value.

Reply:
Can cach  contain `null` as value?

Reply:
Nope, because the `cache` is only populated by the `transform` function!

_Ok, it can be done with reflection, but opt out that usages 😉_

Reply:
So technically yes, _just_ by design not.. hmm.. can we analog annotate the population with notnull?

Reply:
never mind, forgot about the package annotations.